ruledata.go

```diff
@@ -134,24 +133,32 @@ done:
 
 			}
 			if m.Header.Type == syscall.NLMSG_ERROR {
-				log.Println("NLMSG_ERROR\n")
+				error := int32(nativeEndian().Uint32(m.Data[0:4]))
+				if error == 0 {
+					log.Println("Acknowledged!!")
```

Comment:
Also for string like "Acknowledged!!" and "NLMSG ERROR", I would recommend to implement a debug flag and print them in debug flag only.

Such strings are don't give much information to the user.